pandas/tests/reshape/test_tile.py

```diff
@@ -512,6 +512,17 @@ def f():
         tm.assert_numpy_array_equal(
             mask, np.array([False, True, True, True, True]))
 
+    def test_cut_read_only(self):
+        # issue 18773
+        readonly = np.arange(0, 100, 10)
+        readonly.flags.writeable = False
+
+        mutable = np.arange(0, 100, 10)
+
+        one_to_hundred = np.arange(100)
```

Comment:
Not quite...that's really zero to 99 :smile:

Reply:
oh duh lol... my bad I'll update it